doc/source/release-notes/Percona-Server-5.7.18-16.rst

```diff
@@ -49,46 +49,49 @@ to error log. Bug fixed :tdbbug:`23`.
 error output. Bug fixed :tdbbug:`24`.
 
 It was not possible to build |Percona Server| on Debian 9 (stretch) due to
-issues with openssl1.1. Bug fixed :bug:`1702903`.
+issues with OpenSSL 1.1. Bug fixed :bug:`1702903` (upstream :mysqlbug:`83814`). 
 
 Packaging was using the ``dpkg --verify`` command which is not available on
 wheezy/precise. Bug fixed :bug:`1694907`.
 
+Enabling and disabling the slow query log rotation spuriously added the version
+suffix to the next slow query log file name. Bug fixed :bug:`1704056`.
+
 With two client connections to a server (debug server build), the server could
 crash after one of the clients set the global option ``userstat`` and flushed
-the client statistics (``FLUSH CLIENT_STATICTICS``) and then both clients were
+the client statistics (``FLUSH CLIENT_STATISTICS``) and then both clients were
 closed. Bug fixed :bug:`1661488`.
 
 |Percona FT| did not pass cmake flags on to snappy cmake. Bug fixed
 :tdbbug:`41`.  The progress status for partitioned TokuDB table ALTERs was
 misleading. Bug fixed :tdbbug:`42`.
 
-When connecting to the Aurora cluster end point using SSL, SAN (Subject
-Alternative Name) certificates were ignored.  Bug fixed :bug:`1673656` (upstream
-:mysqlbug:`68052`).
+When a client application connecting to the Aurora cluster end point using SSL
+(``--ssl-verify-server-cert`` option), wildcard and :abbr:`SAN (Subject
```

Comment:
In 5.7 `--ssl-verify-server-cert ` is considered deprecated. So I would change
`(--ssl-verify-server-cert option)` to `(--ssl-verify-server-cert / --ssl-mode=VERIFY_IDENTITY option)`